libs/shared/acore-world-model/src/options/creature-icon.ts

```diff
@@ -1,7 +1,7 @@
 import { Option } from '@keira/shared/constants';
 
 export const CREATURE_ICON: Option[] = [
-  { value: '', name: 'None' },
+  { value: null, name: 'None' },
```

Comment:
why?

Reply:
Because '' is different than null
If you put '' the query diff will detect a change, with null no